xinference/api/restful_api.py

```diff
@@ -811,15 +810,13 @@ async def stream_results():
                             prompt, system_prompt, chat_history, kwargs
                         )
                     async for item in iterator:
-                        yield f"data: {json.dumps(item)}\n"
+                        yield dict(data=json.dumps(item))
                 except Exception as ex:
```

Comment:
Client disconnects need to be handled: https://github.com/sysid/sse-starlette/blob/159bbece1a58434d12d67934dc99fab556737247/examples/example.py#L62

Reply:
I have handled all the exceptions including `asyncio.CancelledError`.